avionics/common/include/tasks/restart_camera.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
Turns out header guards are useful in that they give you a hack to mock header files in unit tests, by defining the guard itself - so I'm trying to migrate back to using `#ifndef` header guards; besides it's more standards compliant that way